docs/bokeh/source/docs/user_guide/output/jupyter.rst

```diff
@@ -217,7 +217,7 @@ Trusting notebooks
 ~~~~~~~~~~~~~~~~~~
 
 Depending on the version of the notebook you are using, you may have to
-`trust the notebook <https://jupyter-notebook.readthedocs.io/en/stable/security.html#explicit-trust>`_
+`trust the notebook <https://jupyter-notebook.readthedocs.io/en/4.x/security.html#explicit-trust>`_
```

Comment:
```suggestion
`trust the notebook <https://jupyter-server.readthedocs.io/en/stable/operators/security.html#explicit-trust>`_
```

I believe to suggested link is better, even if the linked information block is the same.